package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "@ginkgo-bioworks/react-json-schema-form-builder",
-  "version": "1.0.6",
-  "description": "React JSON Schema Form Editor",
+  "version": "1.0.7",
```

Comment:
Lets leave the version bump for a separate PR -- we'll bundle in the CHANGELOG.md updates there.